FeaturesProvider/openapi.yaml

```diff
@@ -114,6 +114,288 @@ paths:
                         type:
                           type: string
 
+  /api/extract-job-description:
```

Comment:
Is this openapi served in a UI somewhere? If not, can we have it? Would be much nicer to coordinate in general + we can put it in the demo and show how cool we are in enforcing the stuff 🤓

Reply:
In UI -- no, it exists solely to parse a job description (a link or a text) into various fields of an object. I don't exactly understand what do you mean by "can we have it", like, where? How? Show a "hey look i can wrestle JSONs"?